share/config.fish

```diff
@@ -152,3 +152,15 @@ function . --description 'Evaluate contents of file (deprecated, see "source")'
 		source $argv
 	end
 end
+
+# As last part of initialization, source the conf directories
+# Implement precedence (User > Admin > Vendors > Fish) by basically doing "basename"
+set -l sourcelist
+for file in $configdir/fish/conf.d/* $__fish_sysconfdir/conf.d/* $__fish_datadir/vendor_conf.d/*
```

Comment:
shouldn't `$__fish_sysconfdir/conf.d` itself be allowed to not exist? and all of `$__fish_sysconfdir` for that matter. I've been experimenting with empty /etc, and this seems like it would break that.